neutrons/PyVDrive
# Improve IPython/VDRIVE command console UI

**Description of work.**

- Improve PyVDRive's IPython/VDRIVE-compatible command UI.

- In progress of developing a _system test_ framework for all VDRIVE-compatible commands.

- In progress of prepareing document and manual for release

**To test:**

Test shall be passed on analysis cluster.

1.  [x] 1. Launch PyVDrive to IPython/VDrive console UI
2.  [x] 2. Run: VBIN,IPTS=20280,RUNS=169186, output='/tmp/'
3.  [x] 3. Run: chop, ipts=20717, runs=170464, dbin=30, loadframe=1, output='/tmp'
4.  [x] 4. Run a wrong command:  
   CHOP, IPTS=14194, RUNS=96450, dbin=60,loadframe=1,bin=1,DRYRUN=1"
5.  [x] 5. Run command "vhelp"
6.  [x] 6. Run command "refresh"
7.  [x] 7. Review the table
8.  [x] 8. Plot:
9.  [x] 9. Clear
10.  [x] 10. Plot all:
11.  [x]  11. Check manual at ...

This fixes #138.

This also fixes part of #130.
